src/main.rs

```diff
@@ -246,7 +246,6 @@ fn maybe_disown() {
             .stdin(process::Stdio::null())
             .stdout(process::Stdio::null())
             .stderr(process::Stdio::null())
-            .arg("--no-fork")
```

Comment:
Hm.. I think we still want to force `--no-fork` here, in case the users have set the environment variables and the configuration file to fork. Otherwise, there's a risk of getting a recursion where it forks infinitely.

I'm not sure where the duplicate elements come from though.

Reply:
Duplicate option comes from this line.

I start neovide with `--fork`, and it then restart itself with `--no-fork` + old args.

I tested setting NEOVIDE_FORK variable (to 0 and 1), and both cases works as expected. Setting a variable also works fine.

I think, `is_tty()` returns false for forked process, and we don't need explicit `--no-fork` here.

Reply:
Ah, yes, that makes sense